### PR TITLE
Add example with tf stack

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

## This Pull Request implements

Add example tensorflow software stack provided with thamos advise.

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+name = "pypi-org"
+
+[dev-packages]
+
+[requires]
+python_version = "3.6"
+
+[packages.tensorflow]
+index = "tensorflow-pypi-thoth-station-ninja"
+version = "==2.0.0"
```
Comment (fridex) [inline Pipfile]: Do we need to have it for 2.0.0 specifically?

File: README.md [hunk 2]
```diff
@@ -1,12 +1,79 @@
...
+.. code-block:: console
+
+  cat .thoth.yaml
+
+Alternatively, to generate Thoth's configuration file out of the template run the
+following command:
+
+.. code-block:: console
+
+  thamos config --no-interactive --template thoth_conf_template.yaml
```
Comment (fridex) [inline README.md:37]: The config file expansion based on template is more suitable for OpenShift's s2i. `thamos config` should work just fine for setup based on the local environment.

File: README.md [hunk 3]
```diff
@@ -1,12 +1,79 @@
...
+Finally install the requirements in your environment:
+
+.. code-block:: console
+
+  cat requirements.txt  # check requirements with digests
+  python3 -m venv venv/ && . venv/bin/activate
+  micropipenv install
+
+And finally, run the application (the virtual environment needs to be still
+activated):
```
Comment (fridex) [inline README.md:69]: There was no virtual environment activation following the steps.

File: README.md (hunk 2 above)
Comment (fridex) [reply README.md:37]: Hm, but without the template we don't have any control on what environment the user runs... so let's keep it...

File: Pipfile (hunk 1 above)
Comment (pacospace) [reply Pipfile]: to be as updated as possible with tensorflow?

File: README.md (hunk 3 above)
Comment (pacospace) [reply README.md:69]: `python3 -m venv venv/ && . venv/bin/activate`?

Comment (fridex) [reply README.md:69]: :+1:

File: Pipfile (hunk 1 above)
Comment (fridex) [reply Pipfile]: Yes, also to show that we are capable of resolving large stacks (?)

Comment (fridex) [reply Pipfile]: What about this one? Should we keep it?

Comment (pacospace) [reply Pipfile]: sorry, I will update it with `"*"`

**Review by fridex — APPROVED** [review]
LGTM, thanks :+1:

**Pull request merged**